specutils/io/parsing_utils.py

```diff
@@ -22,24 +23,33 @@ def read_fileobj_or_hdulist(*args, **kwargs):
     Returns:
         an Astropy HDUList
     """
```

Comment:
I noticed the (most recent?) docstrings are using the even more verbose `:class:~astropy.io.fits.HDUList` scheme. To be honest I have not checked or tested if or how they render differently in Sphinx, but unless there are any objections I would go with that form.

Reply:
It doesn't really matter in the technical sense. There are quite a few object contexts you can choose (`:class:`, `:func:`, `:meth:`, etc). They each add a css class to the html in the docs for the reference link. If we had a custom css, we could make them different colors, or differentiate them somehow. When you leave the reference as just `astropy.io.fits.HDULIST`, there's no special css class added.

We haven't been consistent on this really. In most cases, I don't prepend the context because we have no stylesheet that takes advantage of it, and it's not used at all by sphinx except for generating css classes. However, I would not be adverse to starting to be more conscious of it just in case in the future a style sheet takes advantage of it.